src/pages/Markets/Recommended.tsx

```diff
@@ -1,18 +1,281 @@
+        <Box me={4}>
+          <Heading size='md' mb={1}>
+            {title}
+          </Heading>
+          {subtitle && (
+            <Text fontSize='sm' color='gray.500'>
+              {subtitle}
+            </Text>
+          )}
+        </Box>
+        <ChainDropdown
+          chainIds={chainIds}
+          chainId={selectedChainId}
+          onClick={setSelectedChainId}
+          showAll
+          includeBalance
+        />
+      </Flex>
+      {children}
+    </Box>
+  )
+}
+
+export const Recommended: React.FC = () => {
+  const translate = useTranslate()
   const headerComponent = useMemo(() => <MarketsHeader />, [])
+  const assetIds = useAppSelector(selectAssetIds)
+  const [selectedChainId, setSelectedChainId] = useState<ChainId | undefined>()
+
+  const { isLoading: isPortalsAssetsLoading, data: portalsAssets } = usePortalsAssetsQuery()
+
+  const rows = useMemo(
+    () => [
+      {
+        title: 'Most Popular',
+        component: (
+          <AssetsGrid
+            assetIds={assetIds}
+            selectedChainId={selectedChainId}
+            // TODO(gomes): loading state when implemented
+            isLoading={isPortalsAssetsLoading}
+          />
+        ),
+      },
+      {
+        title: translate('markets.categories.trending.title'),
+        subtitle: translate('markets.categories.trending.subtitle'),
+        component: (
+          <AssetsGrid
+            assetIds={assetIds}
+            selectedChainId={selectedChainId}
+            // TODO(gomes): loading state when implemented
+            isLoading={isPortalsAssetsLoading}
+          />
+        ),
+      },
+      {
+        title: translate('markets.categories.topMovements.title'),
+        component: (
+          <AssetsGrid
+            assetIds={assetIds}
+            selectedChainId={selectedChainId}
+            // TODO(gomes): loading state when implemented
+            isLoading={isPortalsAssetsLoading}
+          />
+        ),
+      },
+      {
+        title: translate('markets.categories.recentlyAdded.title'),
+        // TODO(gomes): loading state when implemented
+        component: (
+          <AssetsGrid
+            assetIds={assetIds}
+            selectedChainId={selectedChainId}
+            // TODO(gomes): loading state when implemented
+            isLoading={isPortalsAssetsLoading}
+          />
+        ),
+      },
+      {
+        title: translate('markets.categories.oneClickDefiAssets.title'),
+        component: (
+          <LpGrid
+            assetIds={portalsAssets?.ids ?? []}
+            selectedChainId={selectedChainId}
+            isLoading={isPortalsAssetsLoading}
+          />
+        ),
+      },
+      {
+        title: translate('markets.categories.thorchainDefi.title'),
+        component: (
+          <LpGrid
+            assetIds={assetIds}
+            selectedChainId={selectedChainId}
+            // TODO(gomes): loading state when implemented
```

Comment:
[question] do we want these implemented in this PR or in follow-up?

Reply:
That's on purpose! Loading state *is* implemented, but all are leveraging Portals loading state as a monkey patch currently until others are implemented